src/component/term/TermDetail.tsx

```diff
@@ -80,21 +80,21 @@ export class TermDetail extends EditableComponent<TermDetailProps, TermDetailSta
         const vocabularyName: string = this.props.match.params.name;
         const termName: string = this.props.match.params.termName;
         const namespace = Utils.extractQueryParam(this.props.location.search, "namespace");
-        this.props.loadTerm(termName, {fragment: vocabularyName, namespace});
+        this.props.loadTerm(termName, {fragment: vocabularyName, namespace}).then( () => this.loadValidationResults());
     }
 
     private loadValidationResults = () => {
         (this.props.validationResults && this.props.validationResults[this.props.vocabulary.iri]) ?
-        this.computeScore(this.props.validationResults[this.props.vocabulary.iri].filter(result => result.term.iri === this.props.term?.iri)) : this.setBadgeColor(-1);
-    }
+         this.computeScore(this.props.validationResults[this.props.vocabulary.iri].filter(result => result.term.iri === this.props.term?.iri)) : this.setBadgeColor(-1);
+    };
```

Comment:
Would not 'null' value be meaningful instead of -1 for such cases?

Reply:
Changes committed.